server/middlewares/idvalidator.js

```diff
@@ -0,0 +1,10 @@
+const idValidator = (req, res, next) => {
+  const { id } = req.params;
+  const uuidRegexPattern = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
```

Comment:
@TheDrizzyWay I thought we agreed on using validator js, the custom validation you used can be done using validator Js and I believe it would be neater and more explainable.

Reply:
That's true. I looked through the docs and couldn't find a method for validating uuid, hence i used regex.

Reply:
Yea, but there is a method for that `isUUID(str ) check if the string is a UUID`

Reply:
could you please point me to the reference